pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/with-webkit.nix

```diff
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  lib,
+  telegram-desktop,
+  glib-networking,
+  webkitgtk_4_1,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation {
+  pname = "${telegram-desktop.pname}-with-webkit";
+  version = telegram-desktop.version;
+  nativeBuildInputs = [ makeWrapper ];
+  dontUnpack = true;
+  installPhase = ''
+    mkdir -p $out
+    cp -r ${telegram-desktop}/share $out
```

Comment:
Why do we have to copy `$out/share` over.

Reply:
The idea is that you just specify `telegram-desktop-with-webkit` in systemPackages or wherever and get the shortcut in application menu just like if you specify `telegram-desktop`.

Reply:
`ln -s` would be better?

Reply:
Doesn't seem to work, sadly. I tested it now and it seems things have changed since I last tested and we have to substituted binary path in dbus service. Without that, it launches the non-webkit version via D-Bus activation.